indexer/indexcreator.py

```diff
@@ -40,7 +40,8 @@ def create(self, in_memory=False):
         schema = Schema(verb_form=TEXT(stored=True, sortable=True, analyzer=analyzer),
                         infinitive=TEXT(stored=True, analyzer=analyzer),
                         index_letter=TEXT(stored=True, analyzer=analyzer),
-                        file_path=TEXT(stored=True, sortable=True))
+                        file_path=TEXT(stored=True, sortable=True),
+                        autocompete_sorting=TEXT(stored=True, sortable=True))
```

Comment:
El camp que creem al esquema es diu "autocomplete_sorting" però desprem quan el referenciem l'anomem "autocomplete_sorting".

A l'executar-lo diu:

whoosh.fields.UnknownFieldError: No field named 'autocomplete_sorting' in <Schema: ['autocompete_sorting', 'file_path', 'index_letter', 'infinitive', 'verb_form']>

Reply:
Typo. Arregalt